src/assistant/Search/SearchSubmitFab.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import Fab from 'cozy-ui/transpiled/react/Fab'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import PaperplaneIcon from 'cozy-ui/transpiled/react/Icons/Paperplane'
+import { makeStyles } from 'cozy-ui/transpiled/react/styles'
+import { getFlagshipMetadata } from 'cozy-device-helper'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+const useStyles = makeStyles({
```

Comment:
nit: Make a `FixedFab` to factorize the code with the `AddButton` ([code](https://github.com/cozy/cozy-home/blob/e3767d6358bffc2d80cda7a502ce08eed5353464/src/components/AddButton/AddButton.jsx#L28))

Reply:
je n'irai pas jusque là... Le composant vient directement de la doc cozy-ui donc pas besoin de mutualiser les props je pense. Reste le style par contre avec la petite astuce pour flagship app, à voir 🤔